helm/libero-reviewer/templates/service.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ include "libero-reviewer.fullname" . }}
+  labels:
+    app.kubernetes.io/name: {{ include "libero-reviewer.name" . }}
+    helm.sh/chart: {{ include "libero-reviewer.chart" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+spec:
+  type: {{ .Values.service.type }}
+  ports:
+    - port: {{ .Values.service.port }}
+      targetPort: http
+      protocol: TCP
```

Comment:
I'm assuming we are allowing all TCP traffic (not just http) as we use websockets?

Reply:
`http` here refers to the port name (Kubernetes concept) used in https://github.com/elifesciences/libero-reviewer-formula/pull/1/files/f61c045fbb22015668dae0f6f05f813a2641bcdc#diff-50e335f7f094e74ba8c02f039da00c4fR42 so that you don't have to hardcode `3000` in multiple places; `TCP` is used as opposed to `UDP`.

As far as I understand WebSockets should work over port 80/443 by starting an HTTP(S) connection and using the `Upgrade` header. But if you have a test case we can use this could be verified like for `/graphql`.